music_assistant/server/providers/deezer/manifest.json

```diff
@@ -5,6 +5,6 @@
   "description": "Support for the Deezer streaming provider in Music Assistant.",
   "codeowners": ["@Un10ck3d",  "@micha91"],
   "documentation": "https://github.com/orgs/music-assistant/discussions/1245",
-  "requirements": ["deezer-python==6.0.0", "pycryptodome==3.18.0"],
+  "requirements": ["git+https://github.com/music-assistant/deezer-python-async@convert_to_async", "pycryptodome==3.18.0"],
```

Comment:
I think, we should merge convert_to_async before we merge this PR here in MA. Then we could set a tag on main in deezer-python-async and reference that here in the requirements. This way we ensure that we are only using a tested version of deezer-python-async as a requirement. Otherwise we could run into issues in MA whenever we change something in the other repo, without even realizing it here.

Reply:
Just made the [tag on deezer-python-async](https://github.com/music-assistant/deezer-python-async/releases/tag/v0.1.0). Im now changing the dependency in the manifest and will test if everything still works

Reply:
Yes i just commited the change in deps. 
The [itemmapping issue](https://discord.com/channels/753947050995089438/1004642314968367124/1139145197314379877) is still present. But that a music assistant general error. This pr is good to go from me